src/meta/model/src/iceberg_namespace_properties.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2025 RisingWave Labs
```

Comment:
I don't get why we need this. Is it related with #21400? Isn't that a frontend only view?

Reply:
It is related. `iceberg_namespace_properties` is system table used for JDBC catalog, I think we need this initialization for backup, even though currently, it is an empty table. In case we need this table in the future.

Reply:
> we need this initialization for backup

> Previously, we created the hosted iceberg catalog system related tables on demand

I still don't get it. IIUC, `iceberg_namespace_properties` and `iceberg_tables` are not created in meta. Why we need them for backup?

Reply:
The JDBC catalog skd could help us create these 2 tables if they do not exist (i.e., an empty table), but once we write data to them, we are responsible for maintaining them (including backup).

Reply:
Oh, I finally get it.. So we are talking about meta backend as JDBC catalog for RW usage, which does need to store some metadata. 

RW as JDBC catalog (the user-visible system tables) is also just expose this. I thought it's constructed from some other info..

Makes sense that they need backup.

Reply:
> RW as JDBC catalog (the user-visible system tables) is also just expose this. I thought it's constructed from some other info..

Yes, RW as JDBC catalog is read-only, unless users can access the meta store directly (like self-hosted customers)